modules/local/csv_reader.nf

```diff
@@ -0,0 +1,14 @@
+def CSV_READ(samples) {
```

Comment:
This isn't really a module, but a function. I think this should be in a `lib` directory, rather then the `modules` directory.